developer-docs/upgrade/build_n_deploy.md

```diff
@@ -32,18 +32,26 @@ The following is the list of jobs required to be built and deployed :
-|                         |                           |	                   |                     |  Cert      |   Cert      |
-|                         |                           |                    |                     |  Player    |   Player    |  
+| Knowledge Platform Build | Knowledge Platform Deploy | DataPipeline Build | DataPipeline Deploy | Core Build  | Core Deploy                 |
+|--------------------------|---------------------------|--------------------|---------------------|-------------|-----------------------------|
+| KnowledgePlatform        | Search Service            | Analytics          | AnalyticsAPI        |             | cassandra                   |
+|                          | Learning                  |                    | Dataproducts        | Learner     | learner                     |
+|                          |                           |                    | KafkaIndexer        |             | cert\-registry              |
+| Yarn                     | Yarn                      |                    |                     | Cert        | cert                        |
+|                          | Neo4jDefinitionUpdate     | DataPipeline       | Yarn                | Lms         | lms                         |
+|                          |                           |                    | KafkaSetup          |             | print                       |
+|                          |                           |                    |                     |             | cassandra\-cql insert query |
+|                          |                           |                    |                     |             | content\-service            |
+|                          |                           |                    |                     |             | assessment\-service         |
+|                          |                           |                    |                     |             | Keycloak                    |
+|                          |                           |                    |                     | KnowledgeMW | knowledge\-mw\-service      |
+|                          |                           |                    |                     |             | LogEsUpgrade6xx             |
+|                          |                           |                    |                     |             | oauth                       |
+|                          |
+|                          |                           |                    |                     |             | oauth                       |
+|                          |
+|                          |                           |                    |                     |             | Proxy                       |
+|                          |                           |                    |                     |             | OnboardAPIs                 |
```

Comment:
Many build jobs are missing, mappings to be corrected along with job names to match Jenkins job names exactly. Please work with @kali and get this table right.

Reply:
Be aware, I think you're tagging the wrong "kali" person here :)

Reply:
Oops.. sorry